neurodamus/target_manager.py

```diff
@@ -622,3 +622,43 @@ def __init__(self, cell):
                 # Store a SectionRef to the section at the index specified by v_value
                 self.isec2sec[int(v_value)] = Nd.SectionRef(sec=sec)
             index += 1
+
+
+class TPointList:
```

Comment:
I hope that at some point (soon?) we could replace this with a more Pythonic construct.